### DISCUSSION TITLE
Scan_Type2

### DISCUSSION THREAD

**Discussion by MSAfify** [danielver02/ALPS#170] [Q&A] [answered]
Dear Prof. Kris,
Regarding your previous answer, "For the scan_type=2, the code will take the current value of theta fixed." Does that mean the code will use the initial values of k_perp and k_par to calculate the value of theta? For example, for theta = 75, this means the initial values of k_perp and k_par will be 0.009659 and 0.86193, respectively, where I used this formula: theta=arcsin(k_perp/k) and k =.01.

Many thanks in advance!
Mahmoud

**Comment by kgklein** [comment, marked as answer]
Mahmoud- ALPS will take initial values for the kperp and kpar from the &system namelist (or previous wavevector scans) and calculate theta from those values, and then keep that theta constant, scanning from sqrt(kperp^2+kpar^2) to the final wavevector amplitude specified.
Cheers
Prof. Klein

**Reply by MSAfify** [reply]
Great thanks!
